wolfssl/wolfcrypt/asn_public.h

```diff
@@ -341,7 +346,7 @@ typedef struct CertOidField {
 } CertOidField;
 
 typedef struct CertExtension {
-    const char* oid;
+    char        oid[MAX_OID_STRING_SZ];
```

Comment:
this increases the structure for all extensions from sizeof char* to now be 64 bytes, maybe is not a big deal but would be nice if it could be avoided

Reply:
I agree. The impact of this needs to be carefully reviewed. It could break existing users with limited stack. Please investigate an alternate fix in the wolfTPM CSharp wrapper.

Reply:
I'm putting this into a draft mode, removing the label and investigating a fix in wolfTPM instead.  

Reasoning: Too big of an impact so close to the release.